### DISCUSSION TITLE
Log#1 | 나를 위한 글쓰기 | kciter.so

### DISCUSSION THREAD

**Discussion by giscus** [kciter/kciter.github.io#33] [Announcements]
# Log#1 | 나를 위한 글쓰기 | kciter.so

말(言)이라는 한자를 풀어내면 머리(亠)로 두 번(二) 생각하고 입(口)으로 내보내라는 뜻이라고 합니다. 글을 쓰는 것 또한 말처럼 조심해야 한다고 생각합니다.나의 잘못된 지식이 전파되는 것은 많은 사람에게 피해를 입힐 수 있고 바로잡는 것은 더…

https://kciter.so/posts/log-1/

**Comment by ibcylon** [comment]
로그원
